### PR TITLE
Fixes #8605: Focus on technique name when creating technique

### PR DESCRIPTION
**Pull request by RaphaelGauthier**
https://www.rudder-project.org/redmine/issues/8605

### CODE REVIEW

**Comment by jooooooon** [comment]
Ping @VinceMacBuche

File: builder/index.html [hunk 1]
```diff
@@ -78,6 +78,15 @@ <h3 class="modal-title">Clone Technique</h3>
           </form >
         </script>
         <script src="js/ncf.js" ></script>
+        <script type="text/javascript">
+          ($(function() {
+            console.log(window.self);
```
Comment (VinceMacBuche) [inline builder/index.html]: Can you remove console.log ?

File: builder/index.html [hunk 2]
```diff
@@ -7,7 +7,7 @@
         <script src="js/angular-sanitize.min.js"></script>
         <script src="js/ngToast.min.js"></script>
         <script src="js/angular-drag-and-drop-lists.min.js"></script>
-
+        <script src="js/angular-auto-focus.js"></script>
```
Comment (VinceMacBuche) [inline builder/index.html]: Is there a minified version of this ?

**Comment by RaphaelGauthier** [comment]
PR updated

File: builder/css/custom.css [hunk 3]
```diff
@@ -353,3 +353,17 @@ ul[dnd-list] {
     position: absolute;
     right: 10px;
 }
+
+/* --- IFRAME --- */
+body.iframe .navbar-inverse.navbar-static-top {
+    display: none;
+}
+
+body.iframe .jumbotron h1, .jumbotron .h1 {
```
Comment (VinceMacBuche) [inline builder/css/custom.css]: second entry should be on elem "h1" not class "h1" !!

**Comment by VinceMacBuche** [comment]
Focus only work on first display.

You should try something like that: http://stackoverflow.com/questions/14833326/how-to-set-focus-on-input-field#answer-18295416

**Comment by VinceMacBuche** [comment]
(on the updated answer)

**Comment by RaphaelGauthier** [comment]
PR updated

File: builder/index.html [hunk 4]
```diff
@@ -7,7 +7,7 @@
         <script src="js/angular-sanitize.min.js"></script>
         <script src="js/ngToast.min.js"></script>
         <script src="js/angular-drag-and-drop-lists.min.js"></script>
-
+        <script src="js/angular-auto-focus.min.js"></script>
```
Comment (VinceMacBuche) [inline builder/index.html]: You can now remove that file and that line :)

**Pull request merged**